lib/global_uid/base.rb

```diff
@@ -160,9 +171,9 @@ def self.get_connections
 
     def self.get_uid_for_class(klass)
       with_connections do |connection|
+        server = self.servers.find { |s| connection.current_database.include?(s[:name]) }
```

Comment:
Can you please help me understand what happens in this line?

Reply:
It's retrieving the `server_info` for the connection, which is where the allocations are stored. 

This is removed in #70, where we're iterating over servers, instead of database connections, allowing for a cleaner implementation.